app/controllers/krikri/field_value_reports_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Krikri
+  ##
+  # Handles HTTP requests for Field Value Reports
+  #
+  # @see Krikri::FieldValueReport
+  class FieldValueReportsController < ApplicationController
+    ##
+    # Renders the show view for the field value report, given by a compound key.
+    # The compound key is comprised by the field value report's name
+    # (represented by the route's id param) and the provider's id.
+    def show
+      @field_value_report =
+        Krikri::FieldValueReport.find(params[:id], params[:provider_id])
+
+      respond_to do |format|
+        format.html
```

Comment:
Is this responding with a successful status and an empty response body?  (I'm looking at what I understand to be the corresponding view for HTML.)

Should we get an error instead if we're requesting HTML when only CSV makes sense?